package.json

```diff
@@ -67,7 +67,7 @@
       "git add"
     ],
     "src/**/*.go": [
-      "golangci-lint run -c .golangci.yml --fix",
+      "golangci-lint run -c .golangci.yml --fix src/app/backend/...",
```

Comment:
Adding `src/app/backend/...`, golangci-lint runs we expected.

Reply:
Let's say you changed 20 files. Will it run the command 20 times on the same file set?

Reply:
According to lint-staged documentation [1], file pathes would be passed as list.
[1] https://github.com/okonet/lint-staged#lintstagedrc-example

And in my investigation, I feel it runs once even if we change 20 go files.

Reply:
Sure, then it is good.

Reply:
`golangci-lint` with `--fix` was removed temprarily by #3676.
This issue is separated to #3656 and kept to investigate.